### DISCUSSION TITLE
感觉主线走格子彻底取消有点可惜呢

### DISCUSSION THREAD

**Discussion by aino2333** [share121/inter-knot#5383] [其他]
如题，毕竟hdd是绳匠的本职工作😭个人还是希望能在主线中看到相关设定捏

**Comment by ZionHuang28** [comment]
不是说会看剧情保留吗

**Comment by zhrzhr-zhr** [comment]
好像说不会一刀切，但肯定是会大改的

**Comment by 1luik** [comment]
我也觉得

**Comment by Loxia1121** [comment]
嗯...我个人觉得把战斗和走格子分开点就可以了（个人意见）走格子时专心走格子 即时有怪也让代理人砍掉就好了 等到大boss或者什么深入敌营打小怪+小调查（像简卧底的时候那样）就全程地图 不加走格子 还能试用新角色跑图（说到底就是我觉得走格子走着走着突然要我打怪 打完又要走格子 迟迟打不到boss...有点怪（？）

**Comment by buroro** [comment]
会改，但不是一刀切，慢慢等老米端出新格子把，这个玩法感觉还是挺有开发空间的<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
